### DISCUSSION TITLE
Bug today

### DISCUSSION THREAD

**Discussion by Antoine007** [cssbattle/forum#375] [Bugs reports]
<img width="404" alt="Screenshot 2023-12-22 at 10 00 20" src="https://github.com/cssbattle/forum/assets/12230531/a108bbfa-983a-4a35-82bf-b68903811786">

looks like a perfect match. 80% only though 🤔

**Comment by Antoine007** [comment]
Here is the code:

```
<body bgcolor=92278F><div><p></p><a></a><p></p></div><p u></p><p s>
<style>p{height:30}body{margin:75 65} div{display:flex;border-bottom:solid+30px#FFF579;a{width:100;margin:0 30;border:solid+30px#FFF579;border-bottom:0}p{width:33;border-bottom:solid+30px#FFF579;border-top:solid+60px#FFF579;margin:0}}[u],[s]{position:fixed;margin:-60 120;background:#FFF579}[u]{height:200;width:30}[s]{margin:-90 90;width:100
```

**Comment by SelenIT** [comment]
Your code uses the newest CSS Nesting syntax. The Chrome version that is used to check solutions on the CSSBattle server supports only older version of CSS Nesting spec that required nested selectors to start either with a combinator or with an explicit `&` character. Since your elements inside `div` are all its direct children, you can prepend your nested selectors with a `>` combinator:
```html
<body bgcolor=92278F><div><p></p><a></a><p></p></div><p u></p><p s>
<style>p{height:30}body{margin:75 65} div{display:flex;border-bottom:solid+30px#FFF579;>a{width:100;margin:0 30;border:solid+30px#FFF579;border-bottom:0}>p{width:33;border-bottom:solid+30px#FFF579;border-top:solid+60px#FFF579;margin:0}}[u],[s]{position:fixed;margin:-60 120;background:#FFF579}[u]{height:200;width:30}[s]{margin:-90 90;width:100
```
This version gets 100%.

**Reply by Antoine007** [reply]
I see. weird that I never had the same problem in many days
